lexicons/app/bsky/graph/getRelationships.json

```diff
@@ -0,0 +1,65 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.getRelationships",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Enumerates public relationships between one account, and a list of other accounts",
+      "parameters": {
+        "type": "params",
+        "required": ["actor"],
+        "properties": {
+          "actor": { "type": "string", "format": "at-identifier" },
+          "others": {
+            "type": "array",
+            "maxLength": 30,
+            "items": {
+              "type": "string",
+              "format": "at-identifier"
+            }
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["relationships"],
+          "properties": {
+            "actor": { "type": "string", "format": "did" },
+            "relationships": {
+              "type": "array",
+              "items": {
+                "type": "union",
+                "refs": ["#relationship", "app.bsky.graph.defs#notFoundActor"]
+              }
+            }
+          }
+        }
+      },
+      "errors": [
+        {
+          "name": "ActorNotFound",
+          "description": "the primary actor at-identifier could not be resolved"
+        }
+      ]
+    },
+    "relationship": {
```

Comment:
I think we should move this to `app.bsky.graph.defs` as it's a general description of a relationship & I suspect it may be reused across other lexicons. Thoughts?

Reply:
yup, that makes sense!

Reply:
pushed a change.

having it be in defs does make the "which of the two actors are we talking about" thing a bit more complex; I tried to clarify that with a description, but i'd be open if you want to include both DIDs in each relationship object (bloats the wire size a bit), or rename the `did` field to make it clearer.